tests/tools/collab/test_sync.py

```
@@ -0,0 +1,1976 @@
+        tmp_path / 'first.aiida',
+        sender=backend_one,
+        sender_state=state_one,
+        receiver=backend_two,
+        receiver_state=state_two,
+    )
+    seal_cached_calculation(backend_one, lumi)
+
+    filepath = tmp_path / 'delta.aiida'
+    export = export_full(filepath, state=state_one, backend=backend_one, cursor=None, claim=held)
+
+    with pytest.raises(ConfigurationError, match='collab.computer_map'):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)